assets/javascripts/kitten/components/cards/reward-card/components/title.js

```diff
@@ -22,7 +22,7 @@ export const RewardCardTitle = ({
     )}
```

Comment:
(je peux pas ajouter de commentaire sur le `className` vu que tu l'as pas modifié)

Est-ce que tu crois pas que ça pourrait être intéressant d'avoir :

```
    className={classNames('k-RewardCard__title', className)}
    cssColor={disabled ? COLORS.font2 : COLORS.font1}
```

et supprimer la classe CSS (désormais inutile)
https://github.com/KissKissBankBank/kitten/pull/2188/files#diff-32390b26ffb1381546f838846bf94d434564a3ff6bf36dffb00cb7016e0471f9R42

Pour le `cursor: not-allowed;`, je le verrais plutôt sur la `RewardCard` entière, pour ne pas avoir des zones "vides" au sein de la card où le curseur redevient normal